src/bundle/Resources/public/js/scripts/sidebar/btn/location.edit.js

```diff
@@ -14,17 +15,31 @@
         const showModal = (modalHtml) => {
             const wrapper = doc.querySelector('.ez-modal-wrapper');
             wrapper.innerHTML = modalHtml;
-            wrapper.querySelector('.ez-btn--add-draft').addEventListener('click', addDraft, false);
+            const addDraftButton = wrapper.querySelector('.ez-btn--add-draft');
+            if (addDraftButton) {
+                addDraftButton.addEventListener('click', addDraft, false);
+            }
             [...wrapper.querySelectorAll('.ez-btn--prevented')].forEach(btn => btn.addEventListener('click', event => event.preventDefault(), false));
             $('#version-draft-conflict-modal').modal('show').on('hidden.bs.modal', resetRadioButtons);
         };
 
+        const changeFormActionUrl = () => {
+            if (!form.querySelector('#user_edit_version_info')) {
+                return;
+            }
+            const versionNo = form.querySelector('#user_edit_version_info_version_no').value;
+            const language = btns.filter(btn => btn.checked = true)[0].value;
```

Comment:
![oxmdj7p](https://user-images.githubusercontent.com/12594013/45141204-81d1b000-b1b5-11e8-944f-bd55e5df7235.jpg)

should be `btn.checked === true` event better `btn.checked` and you can use `array.find` so in the end:
`const language = btns.find(btn => btn.checked).value;`

Reply:
![image](https://www.awesomelyluvvie.com/wp-content/uploads/2013/03/Shame.gif)